v3/src/lib/debug.ts

```diff
@@ -16,6 +16,7 @@ export const DEBUG_MAP = debugContains("map")
 export const DEBUG_SAVE = debugContains("save")
 export const DEBUG_STORES = debugContains("stores")
 export const DEBUG_UNDO = debugContains("undo")
+export const DEBUG_PLUGINS = debugContains("plugins")
```

Comment:
This list is alphabetized, so move this ⬆️ . (Given your predilection for alphabetizing lists I figured you'd notice that. 😉)